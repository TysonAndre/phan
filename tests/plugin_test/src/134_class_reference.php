<?php
namespace X\Y\Z;
class Foo{}
class Other{}

namespace A\B\C;
use X\Y\Z\Foo;
// Unconditionally treat Foo::class (ast\AST_CLASS_NAME) as a reference to that class,
// in the same way using a constant from that class (ast\AST_CLASS_CONST) would count as a reference.
$something = Foo::class;
