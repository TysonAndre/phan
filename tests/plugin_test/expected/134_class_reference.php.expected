src/134_class_reference.php:4 PhanUnreferencedClass Possibly zero references to class \X\Y\Z\Other
